src/Elliptic/Systems/Poisson/Equations.hpp

```diff
@@ -187,6 +187,27 @@ struct FirstOrderInternalPenaltyFlux {
       const Scalar<DataVector>& minus_normal_dot_grad_field_exterior) const
       noexcept;
 
+  // This function computes the boundary contributions from Dirichlet boundary
+  // conditions. This data is what remains to be added to the boundaries when
+  // homogeneous (i.e. zero) boundary conditions are assumed in the calculation
+  // of the numerical fluxes, but we wish to impose inhomogeneous (i.e. nonzero)
+  // boundary conditions. Since this contribution does not depend on the
+  // numerical field values, but only on the Dirichlet boundary data, it may be
+  // added as contribution to the source of the elliptic systems. Then, it
+  // remains to solve the homogeneous problem with the modified source.
+  // The first arguments to this function are the boundary contributions to
+  // compute as as not-null pointers, in the order they appear in the
+  // `system::fields_tag`. They are followed by the field values of the tags in
+  // `system::impose_boundary_conditions_on_fields`. The last argument is the
+  // normalized unit covector to the element face.
+  void compute_dirichlet_boundary(
```

Comment:
I can see how these details might not be wanted in the main doxygen. Are you saying that you intend in the future to expand the main doxygen for this struct to include details about this (and other) boundary computation functions?

Reply:
I think I'll rather move all the more general documentation on how to write elliptic numerical fluxes to the `EllipticSystemsGroup` in Doxygen, so this would include the comments on what functions are expected with which arguments. Then this particular struct only needs to document what equations it implements.